app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+import shutil
+
+
+def copy_file(string: str) -> None:
+    file_name = string.split(" ")[1]
```

Comment:
use unpacking 
```
command, source, destination = string.split(" ")
```

Reply:
it's more readable way to extract needed values

Reply:
fixed